include/deal.II/lac/arpack_solver.h

```diff
@@ -76,6 +76,14 @@ extern "C" void dneupd_(int *rvec, char *howmany, int *select, double *d,
  * <code>dnaupd</code> work and also how to set the parameters appropriately
  * please take a look into the ARPACK manual.
  *
+ * @note Whenever you eliminate degrees of freedom using
+ * ConstraintMatrix, you generate spurious eigenvalues and
+ * eigenvectors. If you make sure that the diagonals of eliminated
+ * matrix rows are all equal to one, you get a single additional
+ * eigenvalue. But beware that some functions in deal.II set these
+ * diagonals to rather arbitrary (from the point of view of
+ * eigenvalue problems) values.
```

Comment:
To my mind, the text in step 36 is lengthy and unnecessary. Instead of doing the right thing, setting the diagonal in the mass matrix to one and leaving it zero in the stiffness matrix, the text provides a lengthy discussion about what can go wrong if we don't.

Do I have permission to fix step 36?